cmd/gitserver/server/patch.go

```diff
@@ -300,8 +314,9 @@ func (s *Server) createCommitFromPatch(ctx context.Context, req protocol.CreateC
 
 func cleanUpTmpRepo(path string) {
 	err := os.RemoveAll(path)
+	logger := log.Scoped("cleanUpTmpRepo", "cleans up temp Repo")
 	if err != nil {
-		log15.Info("unable to clean up tmp repo", "path", path, "err", err)
+		logger.Info("unable to clean up tmp repo", log.String("path", path), log.Error(err))
```

Comment:
Same reasoning as above. 

```suggestion
		logger.Warn("unable to clean up tmp repo", log.String("path", path), log.Error(err))
```